### ISSUE TITLE
TST: PY2 doctest INTERNAL ERROR on Appveyor

### ISSUE THREAD

**Issue by pllim** [astropy/astroquery#1035]
```
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\_pytest\main.py", line 100, in wrap_session
INTERNALERROR>     session.exitstatus = doit(config, session) or 0
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\_pytest\main.py", line 138, in _main
INTERNALERROR>     config.hook.pytest_runtestloop(session=session)
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\__init__.py", line 617, in __call__
INTERNALERROR>     return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\__init__.py", line 222, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\__init__.py", line 216, in <lambda>
INTERNALERROR>     firstresult=hook.spec_opts.get('firstresult'),
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\callers.py", line 201, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\callers.py", line 77, in get_result
INTERNALERROR>     _reraise(*ex)  # noqa
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\callers.py", line 180, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\_pytest\main.py", line 161, in pytest_runtestloop
INTERNALERROR>     item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\__init__.py", line 617, in __call__
INTERNALERROR>     return self._hookexec(self, self._nonwrappers + self._wrappers, kwargs)
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\__init__.py", line 222, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\__init__.py", line 216, in <lambda>
INTERNALERROR>     firstresult=hook.spec_opts.get('firstresult'),
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\callers.py", line 201, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\callers.py", line 77, in get_result
INTERNALERROR>     _reraise(*ex)  # noqa
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\pluggy\callers.py", line 180, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\_pytest\runner.py", line 60, in pytest_runtest_protocol
INTERNALERROR>     nodeid=item.nodeid, location=item.location,
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\_pytest\nodes.py", line 390, in location
INTERNALERROR>     location = self.reportinfo()
INTERNALERROR>   File "C:\conda\envs\test\lib\site-packages\_pytest\doctest.py", line 150, in reportinfo
INTERNALERROR>     return self.fspath, self.dtest.lineno, "[doctest] %s" % self.name
INTERNALERROR> AttributeError: 'NoneType' object has no attribute 'lineno'
==================== 19 passed, 13 skipped in 7.27 seconds ====================
```

**Comment by bsipocz** [comment]
Thanks @pllim. I definitely saw this issue somewhere else already, and will look into the details once the astropy releases are out.

**Comment by pllim** [comment]
Looks similar to https://github.com/astropy/astropy/issues/6418#issuecomment-320196796 -- Maybe the solution is to pin `pytest < 3.2` as you recommended back then.

**Comment by bsipocz** [comment]
Oh, ok, so then this should be fixed in 2.0.4, too by https://github.com/astropy/astropy/pull/6430

**Comment by pllim** [comment]
OK closing then.
